From: Alexei Starovoitov <ast@fb.com>
Date: Fri, 27 Oct 2017 10:01:39 -0700
Subject: tcp: remove unnecessary include
Patch-mainline: v4.15-rc1
Git-commit: 5b52a4c3acf5f4b4854d1c3ddc8be8770330a79c
References: bsc#1109837

two extra #include are not necessary in tcp.h
Remove them.

Fixes: 40304b2a1567 ("bpf: BPF support for sock_ops")
Signed-off-by: Alexei Starovoitov <ast@kernel.org>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 include/net/tcp.h |    3 ---
 1 file changed, 3 deletions(-)

--- a/include/net/tcp.h
+++ b/include/net/tcp.h
@@ -45,9 +45,6 @@
 
 #include <linux/seq_file.h>
 #include <linux/memcontrol.h>
-
-#include <linux/bpf.h>
-#include <linux/filter.h>
 #include <linux/bpf-cgroup.h>
 
 extern struct inet_hashinfo tcp_hashinfo;
